src/modules/NewReputation/index.tsx

```diff
@@ -131,6 +129,7 @@ export default function NewReputation() {
     setIsMinting(true);
 
     await wallet.watchTransaction(reputableTxHash);
+    detectContractCalls(ethAddress);
```

Comment:
This returns a promise, you need to `await` here.

Reply:
fixed